install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -796,6 +796,12 @@ data:
   bgp-announce-pod-cidr: {{ .Values.bgp.announce.podCIDR | quote }}
 {{- end}}
 
+{{- if .Values.bgpControlPlane.enabled }}
+  enable-bgp-control-plane: "true"
+  {{- else }}
+  enable-bgp-control-plane: "false"
+{{- end }}
```

Comment:
nit:
```suggestion
  enable-bgp-control-plane: {{ .Values.bgpControlPlane.enabled | quote }}
```

Reply:
IMO, I'd rather keep these blocks "copy and pastable" and other blocks tend to do this if/else.

Reply:
FWIW we use the pattern I suggested in a few places already (e.g. [here](https://github.com/cilium/cilium/blob/1b03f0b4c9dd9f7d7b5886ca563b2c21a4e2745b/install/kubernetes/cilium/templates/cilium-configmap.yaml#L131-L132) and [here](https://github.com/cilium/cilium/blob/1b03f0b4c9dd9f7d7b5886ca563b2c21a4e2745b/install/kubernetes/cilium/templates/cilium-configmap.yaml#L674-L675)) but I'm ok with if/else as well.